services/device-registry-jdbc/src/test/java/org/eclipse/hono/deviceregistry/jdbc/impl/TenantServiceTest.java

```diff
@@ -158,20 +158,21 @@ void testCreateAndFind(final Vertx vertx, final VertxTestContext context) {
                     }));
         });
 
-        final var readManagement = context.checkpoint();
-        create.future().onSuccess(x -> {
-            this.tenantManagement
-                    .readTenant("t1", SPAN)
-                    .onComplete(context.succeeding(result -> {
-                        context.verify(() -> {
-
-                            assertThat(result.isOk())
-                                    .isTrue();
-
-                            readManagement.flag();
-                        });
-                    }));
-        });
+        // TODO: How is this supposed to work?
+        // final var readManagement = context.checkpoint();
```

Comment:
If I uncomment this block of code, it will produce Vert.x checkpoint related exception.

Reply:
This was something I needed to fix on JDBC client side to be more Vert.x compliant.